db/migrate/20240726131027_update_empty_strings_boxes_settings_obo.rb

```diff
@@ -0,0 +1,7 @@
+class UpdateEmptyStringsBoxesSettingsObo < ActiveRecord::Migration[7.1]
+  def up
+    Upvs::Box.find_each do |box|
+      box.update(settings_obo: (box.settings_obo.present? ? box.settings_obo : nil))
```

Comment:
Tu to vies asi urobit aj jednou query co si vypyta prazdny string ale good enough